lib/stingray_docs_internal/generator.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative "../stingray_docs_internal"
```

Comment:
The main file is `lib/stingray_docs_internal.rb`, so we shouldn't require anything inside of _outer_ layer. Do instead:
```ruby
# lib/stingray_docs_internal.rb
...
require_relative 'stingray_docs_internal/generator'
```

Reference: https://github.com/unurgunite/genius-api/blob/master/CONTRIBUTING.md#project-structure